### PR TITLE
Changelog cleanup in preparation for 0.4.17.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by axic** [comment]
Please also sort the typechecker lines (they are not sorted atm).

File: Changelog.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-### 0.4.17 (unreleased)
+### 0.4.17 (2017-09-21)
 
 Features:
- * Support ``pragma experimental "v0.5.0";`` to turn on upcoming breaking changes.
  * Assembly Parser: Support multiple assignment (``x, y := f()``).
- * Code Generator: Added ``.selector`` member on external function types to retrieve their signature.
  * Code Generator: Keep a single copy of encoding functions when using the experimental "ABIEncoderV2".
+ * Code Generator: Partial support for passing ``structs`` as arguments and return parameters (requires ``pragma experimental ABIEncoderV2;`` for now).
```
Comment (axic) [inline Changelog.md:6]: Why is it partial? What is missing?

Comment (chriseth) [reply Changelog.md:6]: decoding

Comment (axic) [reply Changelog.md:6]: Oh right, so perhaps mention that :)

**Pull request merged**